<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="1" column="0">
     <widget class="QFrame" name="VerticalToolbar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>20</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="addFromTreeButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#addFromTreeButton{
background-color: rgba(27, 31, 216, 0.59);
}
QPushButton#addFromTreeButton:hover{
margin-right: 2%;
margin-top: 4%;
background-color: rgba(66, 68, 226, 0.59);
}
QPushButton#addFromTree:pressed{
margin-right: 4%;
margin-top: 8px;
background-color: rgba(66, 68, 226, 0.59);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/rec/img/add_icon.png</normaloff>:/rec/img/add_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="DataBaseButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#DataBaseButton{
background-color: rgba(27, 31, 216, 0.59);
}
QPushButton#DataBaseButton:hover{
margin-right: 2%;
margin-top: 4%;
background-color: rgba(66, 68, 226, 0.59);
}
QPushButton#DataBaseButton:pressed{
margin-right: 4%;
margin-top: 8px;
background-color: rgba(66, 68, 226, 0.59);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/rec/img/sql_icon.png</normaloff>:/rec/img/sql_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="LinesButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#LinesButton{
float right;
border-radius: 5px;
padding: 5px9px;
font-size: 1.2em;
background-color: rgba(27, 31, 216, 0.59);
text shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(27, 31, 216, 0.59);
color: white;
font-family: 'Roboto Slab', serif;
}

QPushButton#LinesButton:hover{
float right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px9px;
font-size: 1.2em;
background-color: rgba(66, 68, 226, 0.59);
text shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(66, 68, 226, 0.59);
color: white;
font-family: 'Roboto Slab', serif;
}
QPushButton#LinesButton:pressed{
float right;
margin-right: 4%;
margin-top: 8px;
border-radius: 5px;
padding: 5px9px;
font-size: 1.2em;
background-color: rgba(81, 83, 225, 0.59);
text shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(81, 83, 225, 0.59);
color: white;
font-family: 'Roboto Slab', serif;
}</string>
       </property>
       <property name="text">
        <string>Lines</string>
       </property>
      </widget>
      <widget class="QPushButton" name="RadarButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#RadarButton{
float right;
border-radius: 5px;
padding: 5px9px;
font-size: 1.2em;
background-color: rgba(27, 31, 216, 0.59);
text shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(27, 31, 216, 0.59);
color: white;
font-family: 'Roboto Slab', serif;
}

QPushButton#RadarButton:hover{
float right;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
padding: 5px9px;
font-size: 1.2em;
background-color: rgba(66, 68, 226, 0.59);
text shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(66, 68, 226, 0.59);
color: white;
font-family: 'Roboto Slab', serif;
}
QPushButton#RadarButton:pressed{
float right;
margin-right: 4%;
margin-top: 8px;
border-radius: 5px;
padding: 5px9px;
font-size: 1.2em;
background-color: rgba(81, 83, 225, 0.59);
text shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(81, 83, 225, 0.59);
color: white;
font-family: 'Roboto Slab', serif;
}</string>
       </property>
       <property name="text">
        <string>Radar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ItemsListButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Items</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="HorizontalToolbar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="handButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#handButton{
background-color: rgba(27, 31, 216, 0.59);
}
QPushButton#handButton:hover{
margin-right: 2%;
margin-left: 2%;
margin-bottom: 2%;
margin-top: 2%;
background-color: rgba(66, 68, 226, 0.59);
}
QPushButton#handButton:pressed{
margin-right: 3%;
margin-top: 5px;
background-color: rgba(66, 68, 226, 0.59);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/rec/img/grab.png</normaloff>:/rec/img/grab.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="playButton">
       <property name="geometry">
        <rect>
         <x>1160</x>
         <y>0</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#playButton{
background-color: rgba(27, 31, 216, 0.59);
}
QPushButton#playButton:hover{
margin-right: 2%;
margin-left: 2%;
margin-bottom: 2%;
margin-top: 2%;
background-color: rgba(80, 241, 118, 0.92);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/rec/img/play.png</normaloff>:/rec/img/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pauseButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1210</x>
         <y>0</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pauseButton{
background-color: rgba(27, 31, 216, 0.59);
}
QPushButton#pauseButton:hover{
margin-right: 2%;
margin-left: 2%;
margin-bottom: 2%;
margin-top: 2%;
background-color: rgba(244, 47, 87, 0.92);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/rec/img/pause.png</normaloff>:/rec/img/pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="Center" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QDockWidget" name="DockWidgetForTree">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>281</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>100</horstretch>
         <verstretch>200</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>119</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="floating">
        <bool>false</bool>
       </property>
       <property name="features">
        <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
       </property>
       <widget class="QWidget" name="dockWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeWidget" name="TreeAddedItems">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerMinimumSectionSize">
            <number>3</number>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>3</number>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Map" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>1201</width>
         <height>551</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>160</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </widget>
       <widget class="QPushButton" name="SetLine">
        <property name="geometry">
         <rect>
          <x>1070</x>
          <y>450</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SetLine</string>
        </property>
       </widget>
       <widget class="QTreeWidget" name="ItemsListWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>181</width>
          <height>281</height>
         </rect>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">QMenu#menuFile:hover{
	background-color: rgba(149, 200, 249, 0.95);
}</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionNETU"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionNO_HELP"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionauthors"/>
   </widget>
   <widget class="QMenu" name="menuHand">
    <property name="title">
     <string>Hand</string>
    </property>
    <addaction name="actionLine"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
   <addaction name="menuInfo"/>
   <addaction name="menuHand"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>50</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionauthors">
   <property name="icon">
    <iconset>
     <normalon>img/plane_115212.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>authors</string>
   </property>
  </action>
  <action name="actionNO_HELP">
   <property name="icon">
    <iconset>
     <normaloff>img/zrk.png</normaloff>img/zrk.png</iconset>
   </property>
   <property name="text">
    <string>NO HELP</string>
   </property>
  </action>
  <action name="actionNETU">
   <property name="text">
    <string>ChangeCursor</string>
   </property>
  </action>
  <action name="actionHand">
   <property name="text">
    <string>Hand</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
